src/main/src/main/java/org/geoserver/system/status/MetricValue.java

```diff
@@ -34,6 +34,8 @@
 
     int priority;
 
+    ValueHolder holder;
```

Comment:
This was removed in https://github.com/geoserver/geoserver/pull/3590/files#diff-1e72fa7cb4d8fce2c903e9686f4533d6L41
What's going on? :-D

Reply:
@aaime  the removal of ValueHolder caused metric values to not be included in json and xml output format, so I put them back